src/components/MarkDown/index.tsx

```diff
@@ -38,7 +37,9 @@ const headingRenderer = ({
 }
 
 const inlineCodeRenderer = ({ children }: { children: ReactNode }) => (
-  <Typography variant="command">{children}</Typography>
+  <Text as="code" variant="code">
```

Comment:
We'll miss a background here no ?

Reply:
@chambo-e yes you're right I forgot it :)